packages/styles/scss/patterns/sub-patterns/content-section/_content-section.scss

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+
+@mixin themed-items {
+  color: $text-01;
+  background: $ui-background;
+}
+
+@mixin content-section {
+  .#{$prefix}--content-section {
+    @include carbon--make-row;
+
+    &__heading {
+      margin-bottom: $carbon--layout-03;
+      @include carbon--type-style('expressive-heading-04', true);
+
+      &__col {
+        @include carbon--make-col-ready;
+        @include carbon--make-col(4, 4);
+
+        @include carbon--breakpoint(lg) {
+          @include carbon--make-col(4, 16);
+        }
+      }
+    }
+    &__children {
+      &__col {
+        @include carbon--make-col-ready;
+        @include carbon--make-col(4, 4);
+
+        @include carbon--breakpoint(lg) {
+          @include carbon--make-col(12, 16);
+        }
+      }
+    }
+
+    padding-top: $carbon--layout-05;
+    padding-bottom: $carbon--layout-05;
+
+    @include carbon--breakpoint('md') {
+      padding-bottom: $carbon--layout-06;
+    }
+    @include carbon--breakpoint('lg') {
+      padding-top: $carbon--layout-05;
+      padding-bottom: $carbon--layout-07;
+    }
+  }
+
+  .#{$prefix}--content-section--g10 {
+    @include carbon--theme($carbon--theme--g10) {
+      @include themed-items;
+    }
+  }
```

Comment:
@annawen1 I noticed you were adding this to your storybook. Should we follow that same pattern here?

Reply:
Take a look at this.

Reply:
So the drupal team wants the option to change the theme through props which is why I think it's fine here?

For the `card` component, I set it through storybook since `card` being used in sections and blocks which will handle the theming